src/components/SearchResults/AlignItemsList.jsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { withStyles, makeStyles } from '@material-ui/core/styles';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import Divider from '@material-ui/core/Divider';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListItemAvatar from '@material-ui/core/ListItemAvatar';
+import Avatar from '@material-ui/core/Avatar';
+import Typography from '@material-ui/core/Typography';
+import aoc from './aoc.jpg';
+
+const useStyles = makeStyles(theme => ({
```

Comment:
i think its better to put style in another file and not inline--i can help you with this tomorrow if u need it